src/clj/prerenderer/core.clj

```diff
@@ -70,10 +74,13 @@
                                                          .deleteOnExit))
                           :start-timeout     5000
                           :wait              false
-                          :noop-when-stopped false}
+                          :noop-when-stopped false
+                          :working-directory "."}
```

Comment:
I'm just wondering, should :working-directory be "." by default or should it be nil and have the old behaviour in that case? What do you think?

Otherwise, is "." cross platform?